lib/src/flutterflow_cli_base.dart

```diff
@@ -6,6 +6,7 @@ import 'package:http/http.dart' as http;
 import 'package:path/path.dart' as path_util;
 
 const kDefaultEndpoint = 'https://api.flutterflow.io/v1';
+bool kIsCli = true;
```

Comment:
I'm not a fan of this global field to be honest...

Initially this package was used just for the CLI. Then we extracted some functions to make them available to be called from Dart. So let's separate concerns this way:
- Keep `flutterflow_cli_base.dart` a generic collection of code used to interact with FlutterFlow API. We can even rename it to `flutterflow_api_client` or something like this.
- Keep CLI-specific logic in `bin/flutterflow_cli.dart`.

Then code in this file should not call `exit`. So let's do this:
- Replace `exit()` calls with `throw`s.
- Wrap the `exportCode()` in `bin/flutterflow_cli.dart` in a try/catch and `exit(1)` when an exception is caught.
- We can keep `stderr.write()` calls as-is I think.

Reply:
Okay, this is a much better idea. Have made the changes.